kinds/gen.go

```diff
@@ -40,6 +40,7 @@ func main() {
 		codegen.BaseCoreRegistryJenny(filepath.Join("pkg", "registry", "corekind"), kindsys.GoCoreKindParentPath),
 		codegen.LatestMajorsOrXJenny(kindsys.TSCoreKindParentPath, codegen.TSTypesJenny{}),
 		codegen.TSVeneerIndexJenny(filepath.Join("packages", "grafana-schema", "src")),
+		codegen.DocsJenny(filepath.Join("docs", "sources", "developers", "kinds", "core"), true),
```

Comment:
Should we do something similar for composable kinds? (I guess so, but wanna confirm) 🤔  cc/ @sdboyer

I guess would make sense to make `DocsJenny` responsible for both, so we could just pass the path til `.../kinds` and let if create `core` and `composable` directories. Any objection? @undef1nd

Reply:
That would be the best, agree.

Reply:
yep yep!

In an ideal world, the jenny itself only decides the filename, and there's a generic jenny pipeline component that we use when defining the overall pipeline to specify its leading path.

But the type system for jennies is too clunky right now to really support that, so what you're describing is :+1:. For now, what's really important is having just one jenny, regardless of whether this is a core or composable kind.